src/native-common/FrontLayerViewManager.tsx

```diff
@@ -134,6 +134,14 @@ export class FrontLayerViewManager {
         return null;
     }
 
+    public getActivePopupId() : string | null {
```

Comment:
>getActivePopupId [](start = 11, length = 16)

Why wasn't this needed in the web implementation?

Reply:
Web is structured differently. FrontLayerViewManager calls RenderDOM.render() by passing a RootView element with an onDismissPopup prop.
In native-common it's RootView that calls into FrontLayerViewManager 

---
In reply to: [159348484](https://github.com/Microsoft/reactxp/pull/404#discussion_r159348484) [](ancestors = 159348484)